src/random/cuda/RngEngine.cuh

```diff
@@ -31,14 +31,14 @@ class RngEngine
 
   public:
     // Construct from state
-    __device__ inline RngEngine(const RngStatePointers& view,
-                                const ThreadId&         id);
+    CELER_FORCEINLINE_FUNCTION RngEngine(const RngStatePointers& view,
+                                         const ThreadId&         id);
 
     // Initialize state from seed
-    __device__ inline RngEngine& operator=(Initializer_t s);
+    CELER_FORCEINLINE_FUNCTION RngEngine& operator=(Initializer_t s);
```

Comment:
I think in general modern compilers are good to trust when to inline a function or not without us directing to force it, so I'd prefer to leave off the additional `__forceinline__` unless we can actually demonstrate in some case that it changes the compiler behavior. Can you change these to simply `inline CELER_FUNCTION`?

Reply:
dorp __forceinline__ and use CELER_FUNCTION instead.